plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerDistributedQueriesWithSnapshotIsolation.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.sqlserver;
+
+public class TestSqlServerDistributedQueriesWithSnapshotIsolation
+        extends AbstractSqlServerDistributedQueries
```

Comment:
let's run a focused test class with just few test methods

Reply:
Which tests should be there? As far as I can see these tests are not taking much longer than previously - these modules still finish < 30 minutes.

Reply:
how much do the new tests take?

Reply:
~ 6 minutes

Reply:
Let's have a dedicaed test methods to keep this under 1 minute (ie startup time)

Reply:
Done.